### ISSUE TITLE
Endless loop when more than one page of github releases

### ISSUE THREAD

**Issue by sconix** [rhysd/changelog-from-release#10]
We started to hit this weird bug that CHANGELOG.md started to get really big and the action takes very long, basically until github actions interrupts it. 

Now by looking at the code it seems that changelog-from-release only checks that is the nextPage 0 but does not use that for the iterative call of list releases so this causes endless loop. 

I hope you could release a fix for this since this is a nice tool, but currently can not be used if there is more than 30 releases.

**Comment by rhysd** [comment]
Is it a public repo? If so, please let me know the URL. I'd like to reproduce the issue.

**Comment by sconix** [comment]
Sadly it is not a public repository and it seems that I do not have a public repository with more than 30 releases.

**Comment by sconix** [comment]
If you are able to make the fix before me I am happy to offer assist in verifying the fix so I can test it out in our private repository before making a release.

**Comment by rhysd** [comment]
I see. Let me confirm two things:

- Is it hosted on github.com? Or is it hosted on your company's GitHub Enterprise instance?
- How many releases in the project? (if you know)

I'll try other some famous OSS which has many releases.

**Comment by sconix** [comment]
Yes hosted on github.com but its private company repository. It now has already 35 releases but the problem started couple releases a go. The issue at least to me is quite clear by looking at the code, the loop is only exited when nextPage is 0 but the pate number is never passed as an option for the list releases call so it keeps fetching the first page again and again. At least that how it manifests for us.

**Comment by rhysd** [comment]
I could reproduce this issue with [electron/electron](https://github.com/electron/electron)

**Comment by rhysd** [comment]
[v2.2.5](https://github.com/rhysd/changelog-from-release/releases/tag/v2.2.5) fixed this issue. Please try it.
